scss/main.scss

```diff
@@ -96,6 +96,25 @@ $grey: #C4C4C4;
   @include box($pink);
 }
 
+#navbar {
+  background: #FFFFFF;
+  display: flex;
+  .nav-link {
+    @include dinpro-text(24px,bold,$lavender);
+    text-transform: uppercase;
+  }
+  .nav-link:hover {
+    transition: .3s ease;
```

Comment:
wait add in the opacity: .7 u had earlier!

Reply:
Lol my b